test/Release/MysqlReleaseGatewayTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 10/08/15
+ */
+
+namespace Clearbooks\LabsMysql\Release;
+
+
+use Clearbooks\Labs\Release\Release;
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DriverManager;
+
+class MysqlReleaseGatewayTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var MysqlReleaseGateway
+     */
+    private $gateway;
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $connectionParams = array(
```

Comment:
Yeah this is a pretty bad place for it, I'll need to implement a better way to handle it at a later date but for now as there is only one test file this was the minimum amount of production code needed to pass the tests.

If you have any suggestions btw it would be much appreciated as setting up db stuff is all new to me, I was thinking of a class for each table? And a base table class other would extend from that would do the connection stuff in the constructor, and maybe pulled db details from a config file perhaps?